src/handlers/inbound/consentRequests/{ID}.ts

```diff
@@ -79,6 +80,21 @@ async function patch (_context: unknown, request: Request, h: StateResponseToolk
   return h.response().code(Enum.Http.ReturnCodes.ACCEPTED.CODE)
 }
 
+/**
+ * Handles an inbound `PUT /consentRequests/{ID}` request
+ */
+async function put (_context: unknown, request: Request, h: StateResponseToolkit): Promise<ResponseObject> {
+  const consentRequesttId = request.params.ID
+
+  const channel = PISPConsentRequestsModel.notificationChannel(consentRequesttId)
+  const pubSub = h.getPubSub()
+  // don't await on promise to resolve, let finish publish in background
+  pubSub.publish(channel, request.payload as unknown as Message)
```

Comment:
Can we please use `setImmediate()` like the above to keep things consistent?

Reply:
Ok fixed